### ISSUE TITLE
Neural Networks are Surprisingly Modular

### ISSUE THREAD

**Issue by sanxing-chen** [STayinloves/paper-note#15]
Authors: Daniel Filan, Shlomi Hod, Cody Wild, Andrew Critch, Stuart Russell
Time: 10 Mar 2020
Note:
- The paper develops a notion of neural network's modularity
- Empirically, the measurement is highly correlated to MLP's learned ability to modeling
  - Trained MLP is more modular than MLP with random weights
  - MLP trained with dropout is more modular than one w/o it
  - Modularity seems to arise mostly on learnable datasets
